SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -747,10 +736,20 @@ int32 USpatialNetDriver::ServerReplicateActors_ProcessPrioritizedActors(UNetConn
 				}
 			}
 
+			// SpatialGDK: Here, Unreal would check (again) whether an actor is relevant. Removed such checks.
+			// only check visibility on already visible actors every 1.0 + 0.5R seconds
+			// bTearOff actors should never be checked
+			if (!Actor->GetTearOff() && (!Channel || Time - Channel->RelevantTime > 1.f))
+			{
+				if (DebugRelevantActors)
+				{
+					LastNonRelevantActors.Add(Actor);
+				}
+			}
 
 			// SpatialGDK - We will only replicate the highest priority actors up the the rate limit.
 			// Actors not replicated this frame will have their priority increased based on the time since the last replicated.
-			if (FinalReplicatedCount < RateLimit)
+			if (FinalReplicatedCount < RateLimit && !Actor->GetTearOff())
 			{
 				bIsRelevant = true;
 				FinalReplicatedCount++;
```

Comment:
@joshuahuburn, with the changes to the replication if statement, incrementing the FinalReplicatedCount here may not give the actual replicated count - (isRelevant but not NetReady, or not TearOff, but isRecentlyRelevant and NetReady). Is it worth moving this counter elsewhere, such as where the replication is performed?